tests/src/JIT/HardwareIntrinsics/X86/Shared/AlternatingBinOpTest.template

```diff
@@ -162,6 +162,8 @@ namespace JIT.HardwareIntrinsics.X86
 
         public void RunBasicScenario_UnsafeRead()
         {
+            TestLibrary.TestFramework.BeginScenario(nameof(RunBasicScenario_UnsafeRead));
```

Comment:
Would it be valuable to have `BeginScenario` take in

```
 [System.Runtime.CompilerServices.CallerMemberName] string memberName = "",
    [System.Runtime.CompilerServices.CallerFilePath] string sourceFilePath = "",
    [System.Runtime.CompilerServices.CallerLineNumber] int sourceLineNumber = 0)
```

parameters?

Reply:
Potentially. There are a few perceivable improvements that could be made to the `TestLibrary.TestFramework` class), but those are probably worth having a deeper discussion and done in a future cleanup (since it is used throughout the CoreCLR tests, and not just used by HWIntrinsics).